### PR TITLE
[Qt] AADPI fix - Enables high-DPI scaling in Qt on supported platforms

### PR DESCRIPTION
**Pull request by sinetek**
This takes care of a Qt issue on more recent versions.

fixes: #710

### CODE REVIEW

**Comment by CaveSpectre11** [comment]
@sinetek can you describe what the symptom of this problem was?

**Comment by sinetek** [comment]
> @sinetek can you describe what the symptom of this problem was?

See bitcoin issue #16254.

File: src/qt/veil.cpp [hunk 1]
```diff
@@ -590,13 +590,12 @@ int main(int argc, char *argv[])
     Q_INIT_RESOURCE(veil);
     Q_INIT_RESOURCE(veil_locale);
 
-    BitcoinApplication app(*node, argc, argv);
 #if QT_VERSION > 0x050100
     // Generate high-dpi pixmaps
     QApplication::setAttribute(Qt::AA_UseHighDpiPixmaps);
 #endif
 #if QT_VERSION >= 0x050600
```
Comment (CaveSpectre11) [inline src/qt/veil.cpp]: Not enough to hold it up, but I would prefer if we start using the syntax `QT_VERSION_CHECK(5,6,0)`, especially considering QT_VERSION is actually built from a QT_VERSION_CHECK call.

**Comment by crypt0-fusi0n** [comment]
@mimirmim Saw this one with QA: Pending... do you mean it's ready for QA?

File: src/qt/veil.cpp [hunk 2]
```diff
@@ -609,6 +608,9 @@ int main(int argc, char *argv[])
     QSslConfiguration::setDefaultConfiguration(sslconf);
 #endif
 
+    BitcoinApplication app(*node, argc, argv);
+
```
Comment (MatWaller) [inline src/qt/veil.cpp]: Remove extra white space here, other than that all good.

Comment (sinetek) [reply src/qt/veil.cpp]: done ✍✍

**Pull request merged**